tests/main/test_integration.py

```diff
@@ -584,6 +568,23 @@ def test_system_states_and_recorded_metadata_with_update_to_file_writer_director
                         ][COMPUTER_NAME_HASH]
                     )
 
+        # test second recording
+        for row_idx in range(4):
+            for col_idx in range(6):
+                with h5py.File(
+                    os.path.join(
+                        expected_recordings_dir,
+                        f"{expected_barcode2}__2020_06_15_141957",
+                        f"{expected_barcode2}__2020_06_15_141957__{chr(row_idx+65)}{col_idx+1}.h5",
```

Comment:
not for future: use a function within `LabwareDefinition` to turn a row/column index into a well name.  It's unnecessarily risky to be re-making these calculations each time (risk of off-by-one errors and other things).  The root cause that Anna tracked down for the mislabeling of graphs in the SDK from image data was due to a bug in implementing one of these conversions instead of just using the `labware_domain_models` package